src/routes/Connect.svelte

```diff
@@ -1,4 +1,14 @@
-<script>
+<script lang="ts">
+    import { changeTitle } from '../services/utils';
+
+    const toggleVisibility = () => {
+        const input = <HTMLInputElement>document.querySelector('#secret-key-input');
+        if (input.type === 'password') {
+            input.type = 'text';
+        } else {
+            input.type = 'password';
+        }
+    };
```

Comment:
I'm sure there's a way to do this by assigning dynamic types? such as `inputType={showSecret ? 'text' : 'password'}`